main.tf

```diff
@@ -1,5 +1,39 @@
-resource "azurerm_resource_group" "rg" {
-    name = "${var.name}-RG"  
-    location = "UK South"
+data "azurerm_client_config" "current" {}
+
+data "azurerm_subscription" "current" {}
```

Comment:
Are these two actually needed?

Reply:
not sure, I thought they had a job... not sure now.

Reply:
I removed it...